### ISSUE TITLE
Config option for Lighting FX

### ISSUE THREAD

**Issue by finalsliver** [mekanism/Mekanism#975]
Would it be possible to add an option to remove all extra particles and lighting from the mod?

In particular I am asking for the ability to alter the config on the client so that when machines begin operations that don't emit light or particles.

Particles seems to be handled by Minecraft natively and I have toggled off the power Cables green filling (though I seem to have left it on when I upgraded to V6.

TL:DR

Please add a config option to disable lighting and particle from machines. It is causing lag when machines are used in bulk for some users. FPS drops from ~120 to ~5
